process_bigraph/emitter.py

```diff
@@ -56,3 +87,539 @@ def query(self, query=None):
             result = self.history
 
         return result
+
+
+class DatabaseEmitter(Emitter):
```

Comment:
Emitters are now just Steps, and should inherit from `Step`

Reply:
Correction: we did make an actual `Emitter` class, which inherits from `Step` and adds a new `query` method: https://github.com/vivarium-collective/process-bigraph/blob/14d32a1e7fc34246f91bfe6a500d55f39ad0fcb0/process_bigraph/emitter.py#L8

Reply:
So I think you are good here